### PR TITLE
Adopt page

### PR DESCRIPTION
**Pull request by JessSumner**
Started to build a page for everything to do with adopting a rat. Only kept one feature which is a stats bar. Stats are fixed numbers and not accurate yet.

### CODE REVIEW

**Comment by JessSumner** [comment]
@MattMSumner and @halogenandtoast here's my first PR! Short and sweet!

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -3,3 +3,4 @@
 @import "custom";
 
 
+
```
Comment (MattMSumner) [inline app/assets/stylesheets/application.css.scss]: How would you feel about removing some of these newlines at the end of this file?

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -99,6 +99,45 @@ body {
...
+    display: block;
+    font-size: 1em;
+    font-weight: normal;
+  }
+}
+
+
 footer {
 
 }
```
Comment (MattMSumner) [inline app/assets/stylesheets/custom.css.scss]: github is complaining that there is no newline at the end of this file. In [sublime you can set] `ensure_newline_at_eof_on_save` to make sure it always adds one if you forget. You don't need to know why this is preferable, but if your interested https://robots.thoughtbot.com/no-newline-at-end-of-file

**Comment by JessSumner** [comment]
@MattMSumner and @halogenandtoast I am putting all my CSS code into the custom.css.scss file. I presume it is worth breaking it up. My idea was to put all the CSS for the adopt page into it's own file adopt.css.scss. Would that be beneficial? If so would an adopt.css.scss file inherit the variables from the custom.css.scss file or would I have to redefine them?

**Comment by MattMSumner** [comment]
@JessSumner that's a great idea! You should do that!

File: app/views/pages/adopt.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<div class="stats">
+  <ul>
+    <li>10<span>Rats available</span></li>
```
Comment (MattMSumner) [inline app/views/pages/adopt.html.erb]: Are these numbers placeholders? What do you think of replacing the numbers with the word placeholder so we don't forget to change them later?

File: app/assets/stylesheets/custom.css.scss (hunk 2 above)
Comment (JessSumner) [inline app/assets/stylesheets/custom.css.scss]: Interesting. I've changed that setting. Thank you.
\n

**Comment by MattMSumner** [comment]
Couple of comments but this LGTM

**Pull request merged**